General assembly meeting minutes 2022-06-29 (#1209)
publiccodenet/about: organization/general-assemblies/minutes-2022-06-29.md

---
Type: Resource
---

# General assembly Minutes, 29th June 2022

In this meeting the following descisions were made:

* The annual report as proposed is accepted to be published provisionally lacking external validation.

> * The policy plan as proposed is adopted. The strategic council should be added to the policy plan and an new plan should be presented at the next general assembly.

Reviewer: ```suggestion
* The policy plan is adopted with the change that the strategic council is added to the first phase. A new plan should be presented at the next general assembly.
```
I think it sounds strange to first say adopted "as proposed" and then say it should be changed. I think it is better to say that it is adopted *with* the change.

Reviewer 2: i like this new wording as well
